scalafmt-tests/src/test/scala/org/scalafmt/util/GitOpsTest.scala

```diff
@@ -1,12 +1,209 @@
 package org.scalafmt.util
 
 import java.io.File
+import java.nio.charset.StandardCharsets
+import java.nio.file.{Files, Paths}
 
-import org.scalatest.FunSuite
+import scala.util._
 
-class GitOpsTest extends FunSuite {
-  test("lsTree") {
-    val files = GitOps().lsTree(AbsoluteFile.userDir)
-    assert(files.exists(x => x.path.endsWith(s".gitignore")))
+import org.scalatest._
+import org.scalactic.source.Position
+
+class GitOpsTest extends fixture.FunSuite {
+
+  import Matchers._
+  import GitOpsTest._
+
+  val root = AbsoluteFile.userDir
+  val dirName = "gitTestDir"
+
+  override type FixtureParam = GitOpsImpl
+
+  // DESNOTE(2017-08-16, pjrt): Create a temporary git directory for each
+  // test.
+  override def withFixture(test: OneArgTest) = {
+    val f = Files.createTempDirectory(dirName)
+    val absFile = AbsoluteFile.fromPath(f.toString).get
+    val ops = new GitOpsImpl(absFile)
+    init(ops)
+    // initial commit is needed
+    val initF = touch("initialfile")(ops)
+    add(initF)(ops)
+    commit(ops)
+    val t = try test.toNoArgTest(ops)
+    finally f.toFile.delete
```

Comment:
Wouldn't that delete it when the jvm exits? Which is fine but not the same as what's happening here (where I delete it after the test finishes). 

It might make the suite run faster I guess (since we aren't deleting while running).